src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -288,9 +290,15 @@ export class MoreInfoDialog extends LitElement {
                   )}
                 ></ha-icon-button-prev>
               `}
-          <span slot="title" .title=${title} @click=${this._enlarge}>
-            ${title}
-          </span>
+          <ha-icon-button
+            slot="navigationIcon"
+            .path=${this.large ? mdiArrowCollapse : mdiArrowExpand}
+            .label=${this.hass.localize(
+              `ui.dialogs.generic.${this.large ? "reduce" : "enlarge"}`
+            )}
+            @click=${this._enlarge}
+          ></ha-icon-button>
+          <span slot="title" .title=${title}> ${title} </span>
```

Comment:
I know this was preexisting, but why are we adding `title` attributes here?  (same for other dialogs)